src/app/api/services/dot-confirmation/dot-confirmation.service.ts

```diff
@@ -1,7 +1,7 @@
 import { Injectable } from '@angular/core';
```

Comment:
Did you test everything where you change the BehaviorSubject?

Reply:
Yes. It was tested.